app/metrics/MovementsMetrics.scala

```diff
@@ -18,27 +18,27 @@ package metrics
 
 import com.codahale.metrics.Timer.Context
 import com.kenshoo.play.metrics.Metrics
-import forms.Choice.AllowedChoiceValues._
+import forms.Choice._
 import javax.inject.{Inject, Singleton}
 import metrics.MetricIdentifiers._
 
 @Singleton
 class MovementsMetrics @Inject()(metrics: Metrics) {
 
   val timers = Map(
-    Arrival -> metrics.defaultRegistry.timer(s"$arrivalMetric.timer"),
-    Departure -> metrics.defaultRegistry.timer(s"$departureMetric.timer"),
-    AssociateDUCR -> metrics.defaultRegistry.timer(s"$associationMetric.timer"),
-    DisassociateDUCR -> metrics.defaultRegistry.timer(s"$disassociationMetric.timer"),
-    ShutMucr -> metrics.defaultRegistry.timer(s"$shutMucr.timer")
+    Arrival.value -> metrics.defaultRegistry.timer(s"$arrivalMetric.timer"),
+    Departure.value -> metrics.defaultRegistry.timer(s"$departureMetric.timer"),
+    AssociateDUCR.value -> metrics.defaultRegistry.timer(s"$associationMetric.timer"),
+    DisassociateDUCR.value -> metrics.defaultRegistry.timer(s"$disassociationMetric.timer"),
+    ShutMUCR.value -> metrics.defaultRegistry.timer(s"$shutMucr.timer")
   )
 
   val counters = Map(
-    Arrival -> metrics.defaultRegistry.counter(s"$arrivalMetric.counter"),
-    Departure -> metrics.defaultRegistry.counter(s"$departureMetric.counter"),
-    AssociateDUCR -> metrics.defaultRegistry.counter(s"$associationMetric.counter"),
-    DisassociateDUCR -> metrics.defaultRegistry.counter(s"$disassociationMetric.counter"),
-    ShutMucr -> metrics.defaultRegistry.counter(s"$shutMucr.counter")
+    Arrival.value -> metrics.defaultRegistry.counter(s"$arrivalMetric.counter"),
+    Departure.value -> metrics.defaultRegistry.counter(s"$departureMetric.counter"),
+    AssociateDUCR.value -> metrics.defaultRegistry.counter(s"$associationMetric.counter"),
+    DisassociateDUCR.value -> metrics.defaultRegistry.counter(s"$disassociationMetric.counter"),
+    ShutMUCR.value -> metrics.defaultRegistry.counter(s"$shutMucr.counter")
   )
 
   def startTimer(feature: String): Context = timers(feature).time()
```

Comment:
We can have second overload method with `Choice` argument and then convert it internally to representation. This would increase confidence in API usage.

Reply:
Fixed